README.md

```diff
@@ -18,6 +18,10 @@ as [Endpoints](https://hilla.dev/docs/lit/guides/endpoints),
 [Reactive Endpoints](https://hilla.dev/docs/lit/guides/reactive-endpoints)
 and [Security](https://hilla.dev/docs/lit/guides/security).
 
+The extension is subdivided into two main artifacts based on the desired front-end framework:
+* `quarkus-hilla` for `Lit` based applications
+* `quarkus-hilla-react` for `React` based applications
```

Comment:
```suggestion
* `quarkus-hilla-react` for `React` based applications (starting with `2.4.1`)
```
Instead of `starting with` also `starting from` possible